### ISSUE TITLE
Potential issue with errorExistsAndShouldBeVisible, $submitted and formfield

### ISSUE THREAD

**Issue by PhilFlash** [formly-js/angular-formly#347]
Hi,
I would like transform an example of angular form (from yearofmoo with ngMessage) in formly.
The angular form is here: https://yearofmoo.github.io/ngMessages/
My formly version is here:
http://jsbin.com/qeniru/4/edit?html,js,output

When the user click on submit, all the fields are checked and possibles errors are displayed
(with errorExistsAndShouldBeVisibleExpression).

My problem:
When I click on submit (when all fields are blanks), all fields display error except two required fields:
these fields are in a fieldGroup.
Why ? I don't understand my problem.

Philippe

**Comment by kentcdodds** [comment]
Ah hah! You're right! I need to set `$scope.options.form = $scope.form` somewhere around [here](https://github.com/formly-js/angular-formly/blob/master/src/directives/formly-field.js#L255)

**Comment by kentcdodds** [comment]
I'll get to it as soon as possible, but if you want to jump on it, I'd appreciate it!

**Comment by PhilFlash** [comment]
```
scope.options.form = scope.form
if (scope.options.key) {
        modelValue = `model['${scope.options.key}']`;
      }
```
Tested and it solves the problem..
Thank you for this quick reply.

**Comment by kentcdodds** [comment]
This has been released in `6.13.1`. Thanks @PhilFlash for reporting it and @benoror for fixing it!
